code/api/src/modules/user/mutations.test.js

```diff
@@ -0,0 +1,61 @@
+import request from 'supertest'
+import express from 'express'
+import graphqlHTTP from 'express-graphql'
+import schema from '../../setup/schema'
+import authentication from '../../setup/authentication'
+
+describe('user mutations', () => {
+  let server;
+
+  beforeAll(() => {
+    server = express();
+
+    server.use(authentication)
+
+    server.use(
+      '/',
+      graphqlHTTP(request => ({
+        schema: schema,
+        graphiql: false,
+        context: {
+          auth: {
+            user: request.user,
+            isAuthenticated: request.user && request.user.id > 0
+          }
+        }
+      })))
+  })
+
+  it('userSignup', async () => {
+    const response = await request(server)
+      .post('/')
+      .send({ query: 'mutation { userSignup(name: "test", email: "test3", password: "test") { id name email } }'})
+      .expect(200)
+      expect(response.body.data.userSignup.email).toEqual('test3')
+
+      const id = response.body.data.userSignup.id;
+
+      await request(server)
+        .post('/')
+        .send({ query: `mutation { userRemove(id: ${id}) { id name email } }`})
+        .expect(200)
+  })
+
+  it('addStyleToUser', async () => {
+      const survey = '[{"classy":5,"artsy":2,"punk":7,"sporty":6,"nature":3}]';
```

Comment:
This is the only format we could get working... hopefully we can get it to work with the actual survey results

Reply:
We can ensure that it's in the right format to send to you.